server/src/main/java/org/apache/druid/query/lookup/LookupModule.java

```diff
@@ -80,7 +81,7 @@ public void configure(Binder binder)
     JettyBindings.addQosFilter(
         binder,
         ListenerResource.BASE_PATH + "/" + LookupCoordinatorManager.LOOKUP_LISTEN_ANNOUNCE_KEY,
-        2 // 1 for "normal" operation and 1 for "emergency" or other
+        LOOKUP_LISTENER_QOS_MAX_REQUESTS // 1 for "normal" operation and 1 for "emergency" or other
```

Comment:
Heh neat, is this documented anywhere I wonder?

Reply:
It wasn't, I added a brief mention of it to the lookup docs